neat/optimization_specialist_neat.py

```diff
@@ -19,7 +18,7 @@
 # Change the enemy here, the winner will be saved in winner_neat_[en].pkl
 # Keep in mind that if you run this file, the winner will be overwritten
 
-#en = 1 # fitness = 95.03
+en = 1 # fitness = 95.03
```

Comment:
how about elevate it to the constant on the top of the file and change the name to `ENEMY_IDX`?

Reply:
sure, I will do this, so it's the same notation for both neat and deap